src/templates/layouts/default.jade

```diff
@@ -30,12 +32,12 @@ html
 				{ name: 'shapes', data: { url: './sprites/shapes.svg' } }
 			]
 
+		//- Application init
+		script(src='#{jqueryCDN}', defer)
+		script(src='scripts/#{buildstamp}index.js', defer)
```

Comment:
Dobrý nápad.

Jen teda ten interval se mi vůbec nelíbí. Lepší by byl `onload` v kombinaci s kontrolou, jestli jQuery nebylo už dříve načtené.

Možná by to podobně šlo aplikovat i na `index.js`.